specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -5904,6 +5952,20 @@
           "items": {
             "$ref": "#/definitions/ExcludedPath"
           }
+        },
+        "compositeIndexes": {
```

Comment:
Adding writable properties to an existing response is considered a breaking change and requires a new api version.
https://microsoft.sharepoint.com/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?e=xEcppo&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw

Reply:
Given this version is in broken status. And all the generated SDKs cannot work now. We need fix this version rather than bring a new version.

Reply:
This is the premise of both my comments really, are you planning to change your service to align with this swagger change or is this swagger change to align with the deployed service?

If your service code/contract isn't changing and this is a fix to your sdk/documentation, then we can allow this through (even though it looks like a breaking change).

Reply:
This swagger is change to align with the deployed service. The service code will not be changed.